.circleci/terraform/load-test/main.tf

```diff
@@ -0,0 +1,23 @@
+terraform {
+  backend "s3" {
+  }
+}
+
+provider "aws" {
+  assume_role {
+    role_arn = var.role_arn
+  }
+}
+
+module "load-test" {
+  source = "github.com/hashicorp/consul/test/load/terraform"
```

Comment:
Currently this will only run whatever load test configuration is in the master branch at `test/load/terraform`.